test/build.sh

```diff
@@ -16,8 +16,9 @@ cmakeConfigure()
     local CMAKEARGS=(..
            "-DCMAKE_Fortran_COMPILER=${fcomp}"
            "-DCMAKE_INSTALL_PREFIX=${idir}"
-           "-DCMAKE_CXX_COMPILER=g++"
-           "-DCMAKE_C_COMPILER=gcc"
+           "-DCMAKE_CXX_COMPILER=CC"
+           "-DCMAKE_C_COMPILER=cc"
```

Comment:
Isn't it a better idea to set the environmental variables `CC` (for C) and `CXX` (for C++). This is what I always did and used to be the recommended way.